program-entrypoint/src/lib.rs

```diff
@@ -3,20 +3,23 @@
 //! For more information see the [`bpf_loader`] module.
 //!
 //! [`bpf_loader`]: crate::bpf_loader
+#![no_std]
 
 extern crate alloc;
 use {
-    alloc::vec::Vec,
-    solana_account_info::AccountInfo,
-    solana_pubkey::Pubkey,
-    std::{
+    alloc::{
+        rc::Rc,
+        slice::{from_raw_parts, from_raw_parts_mut},
+        vec::Vec,
+    },
+    core::{
         alloc::Layout,
         cell::RefCell,
         mem::{size_of, MaybeUninit},
         ptr::null_mut,
-        rc::Rc,
-        slice::{from_raw_parts, from_raw_parts_mut},
```

Comment:
Can we keep `slice::{from_raw_parts, from_raw_parts_mut}` under core instead of alloc?